versions.gradle

```
@@ -6,8 +6,8 @@
 // Version constants
 
 ext {
-    versionCode = 49
-    versionName = '2.3.1'
+    versionCode = 50
```

```suggestion
    versionCode = 49
```
This was already incremented, we are just deciding that next version is 2.4.0 instead of 2.3.1